lib/tasks/npm-task.js

```diff
@@ -37,7 +42,11 @@ module.exports = Task.extend({
     // misplaced console.log
     this.disableLogger();
 
-    return npm(this.command, packages, npmOptions, this.npm).
+    var command = (options.dryRun) ?
```

Comment:
This code will never execute if `options.dryRun` is true...line 26 above will exit the method. You either want to revert this change (lines 45-49) or revert the above (lines 25-28).